templates/patterns/text_featured_media/pattern-text-featured-media.html.twig

```diff
@@ -12,13 +14,39 @@
 {% endif %}
 {% set text = text|render %}
 
-{% if image or video or text %}
+{% set media_position_class = ' ecl-u-order-md-last' %}
+{% if 'right' in variant %}
+  {% set media_position_class = '' %}
+{% endif %}
+
+{% set cta = '' %}
+{% if link %}
+  {% set link_type = 'standalone' %}
+  {% if 'featured' in variant %}
+    {% set link_type = 'cta' %}
```

Comment:
Let's make this type call_to_action.

Reply:
`cta` is a type of link that is defined in components/ec-component-link/ecl-link.html.twig. We can't change it.